pkg/kubectl/cmd/resource/get_test.go

```diff
@@ -980,6 +982,34 @@ func TestGetByFormatForcesFlag(t *testing.T) {
 	}
 }
 
+func TestLabelsColumnWithOutputFlag(t *testing.T) {
+	options := &GetOptions{}
+	testCases := []struct {
+		flags map[string]string
+	}{
+		{flags: map[string]string{"show-labels": "true"}},
+		{flags: map[string]string{"show-labels": "true", "output": "wide"}},
+		{flags: map[string]string{"show-labels": "true", "output": "yaml"}},
+	}
+
+	for i, test := range testCases {
+		f, _, _, _ := cmdtesting.NewAPIFactory()
+
+		buf := bytes.NewBuffer([]byte{})
+		errBuf := bytes.NewBuffer([]byte{})
+
+		cmd := NewCmdGet(f, buf, errBuf)
+		for k, v := range test.flags {
+			cmd.Flags().Lookup(k).Value.Set(v)
+		}
+		if i < 2 {
```

Comment:
why if i < 2, is this necessary ?

Reply:
@shiywang First two test cases should not error out, they're correct cases, but third one should. So I chose to use `if i<?` with `assert.NoError` or `assert.Error` IMHO, maybe it's easier to read. WDYT? :smile_cat:

Reply:
ok, I see, then I suggest you write this way : https://github.com/kubernetes/kubernetes/pull/55930/files#r152906891, is there anything unclear ?